Automattic/pocket-casts-ios
# Playback 2023: prompt user to share when a screenshot is taken

| 📘 Part of: #1142 |
|:---:|

Detects when a user takes a screenshot and displays an alert.

<img src="https://github.com/Automattic/pocket-casts-ios/assets/7040243/033b376a-0a9d-44d9-8f65-de4ce193cfad" width="300">

## To test

1. Make sure you're logged in to an account that has a few episodes listened to this year and last year
2. Go to Profile > Settings > Beta Features > enable `endOfYear`
3. Open stories
4. ✅ Check that the share button is not show for the first story
5. Take a screenshot
6. ✅ You should see an alert and the stories should pause
7. Tap "Not now"
8. ✅ Stories should resume
9. Take another screenshot
10. Tap "Share"
11. ✅ The share sheet should appear
12. Dismiss it
13. ✅ Stories should resume
14. Share the first and the last story by taking a screenshot, tapping Share and saving the image
15. ✅ Check that the stories are saved in Photos
